compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```diff
@@ -329,3 +329,4 @@ error.resource.path.cannot.begin.with.slash=resource path cannot begin with slas
 error.missing.field.match.pattern.member=missing field match pattern member
 error.missing.object.constructor.expression=missing object constructor expression
 error.rest.parameter.cannot.be.included.record.parameter=rest parameter cannot be an included record parameter
+error.annotations.not.allowed.for.tuple.rest.descriptor=annotations not allowed for tuple rest descriptor
```

Comment:
```suggestion
error.annotations.not.allowed.for.tuple.rest.descriptor=annotations are not allowed for tuple rest descriptor
```

Reply:
We had a chat ont this. I'll mention you there.